static/CSS/home.css

```
@@ -4,6 +4,11 @@
     al igual que bloqueamos que se repita y le ponemos el artributo fixed
     para que no se mueva y simule el efecto parallax.
 */
+
+@font-face{
+    font-family: 'K2D';
+    src:url('../fonts/K2D-Regular')
```

@lapy0110 sigue sin funcionar, pero fue mi error en mi review pasado. Estabamos colocando el path mal.
```suggestion
    src:url('../fonts/K2D/K2D-Regular.ttf)
```